web-app/js/plugin/BoxPlot.js

```diff
@@ -370,5 +370,43 @@ BoxPlotView.prototype.submit_job = function (form) {
 
 }
 
+BoxPlotView.prototype.clear_high_dimensional_input = function (div) {
+    RmodulesView.prototype.clear_high_dimensional_input.call(this, div);
```

Comment:
Wouldn't `RmodulesView.prototype.clear_high_dimensional_input.call(this, div);` work?